pyoracc/test/model/test_corpus.py

```diff
@@ -25,19 +25,12 @@
 from ..fixtures import tiny_corpus, sample_corpus, whole_corpus
 
 
-slow = pytest.mark.skipif(
-    not pytest.config.getoption("--runslow"),
-    reason="need --runslow option to run"
-)
-
-
```

Comment:
The [latest docs](https://docs.pytest.org/en/latest/example/simple.html#control-skipping-of-tests-according-to-command-line-option) show how to achieve the same effect with non-deprecated syntax. This seems to have changed some time between version 3.0 and 3.4 of pytest. The "new way" might not work with older versions (like 2.9.0) which we want to test, though.

I'm not sure how useful the `slow` marker is - @raquel-ucl, any thoughts?

Reply:
Thanks for the review! I just removed it because @jayanthjaiswal said that was a reasonable solution.

Where does the test requirement for 2.9 come from, by the way? I couldn't find any information other than it fixed some issue with the build, but as far as I can see with #79, pytest 4.6.2 works fine.

Reply:
It looks like the new-style mark requires pytest 3.0.7 or later. 2.9.2 fails.